teacher.rb

```diff
@@ -0,0 +1,18 @@
+require 'person'
+
+# teacher.rb
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission:)
```

Comment:
- [ ] The highligted line of code should be like the code snippet below as currently your line 6 is throwing up an error
- code snippet
```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
```